src/test/java/com/uwetrottmann/tmdb2/TmdbTestSuite.java

```diff
@@ -76,9 +87,55 @@ public static void initializeEnvironment() throws IOException, ParseException {
         unauthenticatedInstance = new BaseTestCase.TestTmdb(API_KEY);
         authenticatedInstance = new BaseTestCase.TestTmdb(API_KEY);
 
+        initializeData();
+
+        initialized = true;
+    }
+
+    public static void initializeData() throws ParseException, IOException, InterruptedException {
```

Comment:
This method is designed to run whatever you are trying to test. And determines with the help of some custom made annotation if it has to initialize the Account and Guest data.